.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -26,7 +26,7 @@ If the support paths suggested above still do not result in a resolution, please
   - OS:
   - Node.js version:
   - npm version:
-  - `` version:
+  - `undefined` version:
```

Comment:
@sofisl I think I need to add something to package.json or .repo-metadata.json.  Do you know what's missing?